Azure/azure-cli
# [App Service] az logicapp deployment source config-zip: Add command to deploy logicapp from zip

**Description**

Add `az logicapp deployment source config-zip` command

**Testing Guide**

Create a logicapp.

`az logicapp deployment source config-zip -n {} --src {path_to_zip}` to deploy zip to logicapp.
---

This checklist is used to make sure that common guidelines for a pull request are followed.
